JsonPreprocessor/CJsonPreprocessor.py

```diff
@@ -707,6 +708,11 @@ def __handleListElements(sInput : str) -> str:
         for line in sJsonData.splitlines():
             if line == '' or line.isspace():
                 continue
+            try:
+                listDummy = shlex.split(line)
+            except Exception as error:
+                raise Exception(f"\n[shlex exception]\n {str(error)} \nin line: '{line}'")
```

Comment:
Hi Son,

the error message part "[shlex exception]" was only intended to be an example in my Python example code. No need to take over this in the code of the JsonPreprocessor. No need to tell the users the name of the parser who causes the error message. Please remove "[shlex exception]" ;-)

Reply:
Hello Holger,

Thank you, I will update!

Thank you,
Son